docs/admin/admin-ui/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+tags:
+- installation
```

Comment:
We usually add tags that also help set the context. In here, you can add `administration` and `admin-ui` to all the pages.

Reply:
done